groups/bsl/bslstl/bslstl_optional.h

```diff
@@ -277,152 +266,100 @@ struct Optional_RemoveCVRef {
         typename bsl::remove_reference<TYPE>::type>::type type;
 };
 
-
-template <class TYPE, class OPT_TYPE>
-struct Optional_ConvertsFrom
-: bsl::integral_constant<
-      bool,
-      bsl::is_convertible<const OPT_TYPE&, TYPE>::value ||
-          bsl::is_convertible<OPT_TYPE&, TYPE>::value ||
-          bsl::is_convertible<const OPT_TYPE, TYPE>::value ||
-          bsl::is_convertible<OPT_TYPE, TYPE>::value ||
-          Optional_IsConstructible<TYPE, const OPT_TYPE&, false>::value ||
-          Optional_IsConstructible<TYPE, OPT_TYPE&, false>::value ||
-          Optional_IsConstructible<TYPE, const OPT_TYPE, false>::value ||
-          Optional_IsConstructible<TYPE, OPT_TYPE, false>::value> {
-    // This metafunction is derived from 'bsl::true_type' if 'TYPE' can be
-    // converted from an 'OPT_TYPE', and from 'bsl::false_type' otherwise.
-};
+// As in 'std' implementation, if the 'TYPE' converts from any value category
+// of an optional type 'OPT_TYPE', we consider it convertible from that
+// optional type.
+#define BSLSTL_OPTIONAL_CONVERTS_FROM(TYPE, OPT_TYPE)                         \
```

Comment:
I think it would make sense to add the `_V` to all macros that evaluate to a `bool`

Reply:
I'm not all that keen on adding _v to the rest of the macros. The only reason I added the _v to the is_constructible and is_assignable one is because their names resemble the standard traits and i thought it might be useful to know the macro evaluates to value member of the trait. I don't see any justification to add it to the other ones, plus it make sthe already long names even longer. I think i'd prefer to remove the _v I added than to modify the remaining traits. WDYT ?